specification/electronic-prescription-service-api.yaml

```diff
@@ -70,12 +70,19 @@ info:
     - it is available in integration and production
     - we might make breaking changes, but only if we cannot avoid it, and we will give advance notice
     
-    #### Tertiary care medication workflow
-    This API does not support tertiary care medication workflow.
-    
     ### Dispensing API
-    Download and the mark as dispensed dispensing flows for the EPS FHIR API are in the sandbox environment only, withdraw and return are currently only referenced in this spec.
-    
+    This API supports two types of dispensing medication workflows
+
+    #### Community pharmacy workflow
+    In a community pharmacy workflow, prescriptions can be releaased (downloaded) from Spine and updated and dispensed by the pharmacy
+    - it is available for testing in the integration environment, but not for production use
+    - we might make breaking changes, but only if we cannot avoid it, and we will give advance notice
+
+    #### Homecare dispenser workflow
+    In a homecare dispenser workflow, prescriptions can be releaased (downloaded) from Spine and updated and dispensed by the dispenser
```

Comment:
is there any reason to duplicate this? we can just say `In both a community pharmacy and homecare workflow, ...`